web_client/src/components/ControlPanel.vue

```diff
@@ -411,7 +411,10 @@ export default {
                           :decision="decision"
                         />
                         <div
-                          v-if="currentViewData.scanDecisions.length === 0"
+                          v-if="
+                            currentViewData?.scanDecisions === undefined
+                              || currentViewData?.scanDecisions?.length === 0
```

Comment:
```suggestion
                            !currentViewData?.scanDecisions?.length
```

Is this equivalent?

Reply:
Yes, thanks for the short version!

Reply:
Actually, I'm discovering that neither works; we cannot use the `?` operator like this within the template. https://github.com/vuejs/vue/issues/11088.

Reply:
If we did want to use that syntax, we could throw it into a computed property and use that in the template.